js/main.js

```diff
@@ -26,9 +28,18 @@ class Products {
             // подсчет суммы всех товаров
             this.summa +=product.price;
             this.container.insertAdjacentHTML('beforeend',product.render());
-           // this.container.insertAdjencentHTML('beforeend',product.render());
+
```

Comment:
_render(){
        for(let data of this.data){
            const product = new ProductItem(data);
            this.products.push(product);
            this.container.insertAdjacentHTML('beforeend',product.render());
        }
    }